src/ontology/uberon-edit.obo

```diff
@@ -223118,6 +223129,14 @@ intersection_of: UBERON:6000021 ! insect abdominal segment
 intersection_of: part_of UBERON:6003023 ! insect adult abdomen
 relationship: part_of UBERON:6003023 ! insect adult abdomen
 
+[Term]
+id: UBERON:6003039
+name: dorsal trunk of insect trachea
+def: "Longitudinal tracheal branch of the insect tracheal system, dorsal to the lateral trunk." [FBbt:00005043]
+synonym: "dorsal trunk" BROAD SENSU [NCBITaxon:50557]
+xref: FBbt:00003039
```

Comment:
Please note that mappings to FBbt are now managed on the FBbt side. They are transferred to Uberon as a SSSOM file, from which cross-references are derived during the build process and inserted into the released ontology.

I’ll add this mapping to FBbt, you may remove it from here (otherwise it will be removed next time the mappings are updated in Uberon).